api/comment_threads.rb

```diff
@@ -54,10 +54,15 @@
   filter_blocked_content params["body"]
   thread.update_attributes(params.slice(*%w[title body pinned closed commentable_id group_id thread_type]))
 
+  if params["user_id"] and value_to_boolean(params["read"])
+    user = User.only([:id, :username, :read_states]).find_by(external_id: params["user_id"])
+    user.mark_as_read(thread) if user
```

Comment:
@wajeeha-khalid.  Agreed we should not add in the exception.  And it seems the comment service doesn't have a logging mechanism for logging the unexpected (e.g. user id is passed but user can't be found).  I'll document this for future comment service improvements.  Thanks.